packages/form-state-manager/src/files/use-field-array.ts

```diff
@@ -79,7 +79,7 @@ const useFieldArray: UseFieldArray = ({ name, initialValue }) => {
   };
   return {
     fields,
-    meta
+    meta: { ...meta, length: internalValue.length }
```

Comment:
according to docs https://final-form.org/docs/final-form/types/FieldState#length

It should be for all fields.

Reply:
Well, that an `useField` attribute then. And affects non-field array fields as well.

Reply:
It should be calculated in managerApi.

Reply:
Yep

Reply:
We should fix it in another issue